pkg/cmd/admin/router/router.go

```diff
@@ -158,7 +166,17 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		}
 	}
 
-	ports, err := app.ContainerPortsFromString(cfg.Ports)
+	// For host networking, use the default ports 80/443. We don't have
+	// config options as yet to specify what the externally exposed
+	// ports are.
+	ports, err := app.ContainerPortsFromString(defaultPorts)
```

Comment:
Not sure what you meant? But the next block of code below this handles the cfg.Ports if the host network stack is disabled (container network is used):  
```!cfg.HostNetwork``` then ```ports, err = app.ContainerPortsFromString(cfg.Ports)```.
See: https://github.com/ramr/origin/blob/router-host-nw/pkg/cmd/admin/router/router.go#L174